apps/utils/db-query-checker/src/csv_log.rs

```diff
@@ -134,7 +134,7 @@ mod tests {
         // Write out a fake row and check the buffer.
         output_csv_row(
             csv_serializer,
-            DatabaseQuery::SelectBeneRecordsByBeneIds,
+            DatabaseQuery::BeneRecordsByBeneIds,
```

Comment:
Not immediately seeing why this changed (or why it changed as a result of this PR)

Reply:
These changes resulted from suggestions made by [clippy](https://doc.rust-lang.org/stable/clippy/index.html), specifically erroring on [Enum Variant Names]( https://rust-lang.github.io/rust-clippy/master/index.html#enum_variant_names).